flare/mgp/otf.py

```diff
@@ -18,7 +17,7 @@
     qe_input_to_structure, parse_qe_forces
 from flare import output
 
-class MFFOTF(OTF):
+class MGPOTF(OTF):
```

Comment:
We now have three otf.py files in the repo -- mgp/otf, which inherits from flare/otf, and ase/otf, which is separate. If there are common methods to all three, we should try to import them from one place to minimize duplication.